examples/nidaqmx_analog_input/measurement.py

```diff
@@ -46,12 +49,7 @@ def measure(pin_name, sample_rate, number_of_samples):
         sample_rate,
```

Comment:
def measure is missing type annotations.
```
measurement.py:71: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
```

Reply:
I will take this in a new PR.